experimental/babel-preset-env-standalone/src/available-plugins.js

```diff
@@ -0,0 +1,13 @@
+import { availablePlugins } from "@babel/standalone";
+
+const notIncludedPlugins = {
+  "transform-new-target": require("@babel/plugin-transform-new-target"),
+};
+
+Object.keys(notIncludedPlugins).forEach(pluginName => {
+  if (!availablePlugins[pluginName]) {
+    availablePlugins[pluginName] = notIncludedPlugins[pluginName];
```

Comment:
The proper way of doing this is the `registerPlugin` method. Please use that instead :)

Maybe it's better to add a `registerPluginUnlessAlreadyRegistered` method (or something like that) to avoid directly messing with `availablePlugins`.